src/main/java/com/keeper/homepage/domain/merit/dto/response/SearchMeritLogListResponse.java

```diff
@@ -0,0 +1,37 @@
+package com.keeper.homepage.domain.merit.dto.response;
+
+import com.keeper.homepage.domain.merit.entity.MeritLog;
+import jakarta.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Builder
+@AllArgsConstructor(access = AccessLevel.PROTECTED)
+public class SearchMeritLogListResponse {
+
+  private long id;
+  private LocalDateTime giveTime;
```

Comment:
```suggestion
@JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
private LocalDateTime giveTime;
```

`LocalDateTime` 양식 지정해주는 건 어떨까요?

Reply:
수정했습니다~ 감사합니다